examples/reference-entity-model/entities/Customer.entity.json

```diff
@@ -12,7 +12,8 @@
       "primaryKey": "customerId",
       "properties": {
         "customerId": {
-          "datatype": "integer"
+          "datatype": "integer",

```

Comment:
Can you add it to birthDate as well? That's the kind of property that is very likely to be sortable.